src/runners/rfqm_meta_transaction_consumer_runner.ts

```diff
@@ -65,6 +68,72 @@ if (require.main === module) {
     })().catch((error) => logger.error(error.stack));
 }
 
+/**
+ * Returns true if the worker has a sufficient balance to trade
+ * @param accountAddress the EOA address
+ * @param accountBalance the EOA address balance
+ * @param gasPriceBaseUnits fast gas price (wei)
+ * @returns true if the worker has sufficient balance
+ */
+export function workerHasEnoughBalance(
```

Comment:
these are awesome! could we put `workerHasEnoughBalance` `workerHasNoPendingTransactionsAsync` and `isWorkerReadyAndAbleAsync` in a new file called `rfqm_worker_balance_utils` or something? Then we can change the merge base to `master` and get this checked in without being blocked by the other commit

Reply:
Done!